packages/odyssey-react/src/components/Button/Button.test.tsx

```diff
@@ -39,6 +39,20 @@ describe("Button", () => {
     expect(getByRole(button)).toHaveAttribute("disabled");
   });
 
+  it("renders an icon", () => {
+    render(
+      <Button
+        icon={
+          <svg>
+            <title>test</title>
+          </svg>
+        }
+      />
+    );
+
+    expect(screen.getByTitle("test").parentElement).toBeVisible();
```

Comment:
+1 adding a test for a top level API like this, if we move to providing icons through children we could probably remove